### PR TITLE
feat(kselect, kmultiselect): dropdown footer text position prop [KHCP-6007]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Ticket: https://konghq.atlassian.net/browse/KHCP-6007

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: docs/components/multiselect.md [hunk 1]
```diff
@@ -221,6 +221,18 @@ Adds informational text to the bottom of the dropdown options which remains visi
 <KMultiselect dropdown-footer-text="Dropdown footer text" :items="items" />
 ```
 
+### dropdownFooterTextPosition
+
+By default, element containing text you pass through `dropdownFooterText` will remain visible even if the content is scrolled. You can use this prop should you need to change that. Accepted values: `sticky`(default) and `static`.
```
Comment (jillztom) [inline docs/components/multiselect.md:226]:
```suggestion
By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible. If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. 

Accepted values: `sticky` (default) and `static`
```

File: docs/components/select.md [hunk 2]
```diff
@@ -223,6 +223,18 @@ Adds informational text to the bottom of the dropdown options which remains visi
 <KSelect dropdown-footer-text="Dropdown footer text" :items="items" />
 ```
 
+### dropdownFooterTextPosition
+
+By default, element containing text you pass through `dropdownFooterText` will remain visible even if the content is scrolled. You can use this prop should you need to change that. Accepted values: `sticky`(default) and `static`.
```
Comment (jillztom) [inline docs/components/select.md:228]:
```suggestion
By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible even if the content is scrolled. If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. This ensures the footer text is visible only when you scrolled to the bottom of the list. 

Accepted values: `sticky` (default) and `static`
```

File: docs/components/multiselect.md [hunk 3]
```diff
@@ -221,6 +221,20 @@ Adds informational text to the bottom of the dropdown options which remains visi
 <KMultiselect dropdown-footer-text="Dropdown footer text" :items="items" />
 ```
 
+### dropdownFooterTextPosition
+
+By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible even if the dropdown content is scrolled. If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. This ensures the footer text is visible only when you scrolled to the bottom of the list.
```
Comment (adamdehaven) [inline docs/components/multiselect.md:226]:
```suggestion
By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible even if the dropdown content is scrolled. 

If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. This ensures the footer text is visible only when the user scrolls to view the bottom of the list. 
```

File: docs/components/select.md [hunk 4]
```diff
@@ -223,6 +223,20 @@ Adds informational text to the bottom of the dropdown options which remains visi
 <KSelect dropdown-footer-text="Dropdown footer text" :items="items" />
 ```
 
+### dropdownFooterTextPosition
+
+By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible even if the dropdown content is scrolled. If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. This ensures the footer text is visible only when you scrolled to the bottom of the list.
```
Comment (adamdehaven) [inline docs/components/select.md:228]:
```suggestion
By default, the dropdown footer text will be stuck to the bottom of the dropdown and will always be visible even if the dropdown content is scrolled. 

If you want to override the behaviour and have the footer text at the end of the dropdown list, use the value `static`. This ensures the footer text is visible only when the user scrolls to view the bottom of the list. 
```

**Comment by adamdehaven** [comment]
Other than the suggested rewording, everything else looks good to me. 

Please provide @helen-tang27 with the deploy preview URL so she can test before merging.

**Comment by adorack** [comment]
Huh, the static text looks good to me, but the first (sticky) example doesn't seem to be sticky for me:

image

Helen says that it works fine for her -- is it just me?

**Comment by adorack** [comment]
I don't see any console errors or obvious CSS issues.  Let me know if I can help debug.

**Comment by portikM** [comment]
> Huh, the static text looks good to me, but the first (sticky) example doesn't seem to be sticky for me:
> 
> image
> 
> Helen says that it works fine for her -- is it just me?

Hmm what browser are you using?

**Comment by adorack** [comment]
Firefox on Linux.

**Comment by adorack** [comment]
That works, thanks!

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**